src/PaginationResult.php

```diff
@@ -64,6 +64,16 @@ public function getIterator()
      */
     public function count()
     {
-        return $this->records instanceof \Countable || is_array($this->records) ? count($this->records) : 0;
+        // @codeCoverageIgnoreStart
+        if (
+            !$this->records instanceof \Countable
+            && !is_array($this->records)
+            && version_compare(PHP_VERSION, '7.2.0', '>=')
+        ) {
+            // PHP: rfc:counting_non_countables https://wiki.php.net/rfc/counting_non_countables
+            trigger_error('count(): Parameter must be an array or an object that implements Countable', E_USER_WARNING);
```

Comment:
This message is copied from here: 

[Online PHP editor | output for jY1aj](https://3v4l.org/jY1aj)